core/src/main/java/nl/nn/adapterframework/http/cxf/SOAPProviderBase.java

```diff
@@ -292,4 +307,18 @@ private XmlBuilder getMimeHeadersXml(Iterator<MimeHeader> mimeHeaders) {
 		}
 		return xmlMimeHeaders;
 	}
+	
+	private String findAction(int start, String contentType) {
```

Comment:
You could determine the start index in this method
`contentType.indexOf("action=")`